xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEBuffer.cpp

```diff
@@ -310,7 +310,8 @@ bool CActiveAEBufferPoolResample::ResampleBuffers(int64_t timestamp)
         }
 
         // pts of last sample we added to the buffer
-        m_lastSamplePts += (in->pkt->nb_samples-in->pkt_start_offset) * 1000 / m_format.m_sampleRate;
+        m_lastSamplePts +=
+            (in->pkt->nb_samples - in->pkt_start_offset) * 1000 / m_inputFormat.m_sampleRate;
```

Comment:
nice finding. maybe use in->pkt->config.sample_rate instead of m_inputFormat.m_sampleRate. that would make it clear that all values belong to the in-buffer

Reply:
Has changed it to use `in->pkt->config.sample_rate` and match more to the other values there, thanks.